drivers/watchdog/Kconfig.dw

```diff
@@ -9,3 +9,21 @@ config WDT_DW
 	depends on DT_HAS_SNPS_DESIGNWARE_WATCHDOG_ENABLED
 	help
 	  Synopsys DesignWare Watchdog driver.
+
+if WDT_DW
+
+config WDT_DW_PAUSE_BY_DBG
+	bool "pause watchdog counter while debugging"
```

Comment:
Please start with uppercase (for consistency).

Reply:
updated